google/api/annotations.proto

```diff
@@ -69,21 +81,16 @@ extend google.protobuf.FieldOptions {
   // See `google/api/resources.proto`.
   string resource_type = 1055;
```

Comment:
Can this be changed to `resourece_reference` to be consistent with the resource set definition?